### PR TITLE
support for csv export for ELAS

### PR DESCRIPTION
**Pull request by dsnyder-fairway**

### CODE REVIEW

File: webapp/src/main/webapp/src/app/csv-export/csv-builder.service.ts [hunk 1]
```diff
@@ -482,12 +500,17 @@ export class CsvBuilder {
...
       .withMigrantStatus(getExam)
       .with504Plan(getExam)
-      .withIep(getExam)
-      .withLimitedEnglish(getExam)
-      .withEthnicity(getExam, ethnicities);
+      .withIep(getExam);
+    if (this.showLep)
+      studentContext = studentContext.withLimitedEnglish(getExam);
+    if (this.showElas)
+      studentContext = studentContext.withElas(getExam)
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/csv-export/csv-builder.service.ts]: nit: needs `{}`

File: aggregate-service/src/main/resources/application.yml [hunk 2]
```diff
@@ -90,4 +90,4 @@ reporting:
 
   # english-learners should be lep or elas
   english-learners:
-    - lep
\ No newline at end of file
+    - elas
```
Comment (dsnyder-fairway) [inline aggregate-service/src/main/resources/application.yml]: moved to elas as we are doing this for CA in our SOW assumption - ASSUMN2 - 10

Not sure if we want to make the default lep and provide documentation for CA

File: webapp/src/main/webapp/src/app/csv-export/csv-builder.service.ts (hunk 1 above)
Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/csv-export/csv-builder.service.ts]: done

File: aggregate-service/src/main/resources/application.yml (hunk 2 above)
Comment (malaffoon) [reply aggregate-service/src/main/resources/application.yml]: I would ask that we make the default what CA wants.

File: report-processor/src/main/resources/application.sql.yml [hunk 3]
```diff
@@ -190,6 +191,12 @@ sql:
         (CASE WHEN e.section504 = 1 THEN 'Yes' when e.section504 = 0 THEN 'No' ELSE '' END),
         (CASE WHEN e.iep THEN 'Yes' ELSE 'No' END),
         (CASE WHEN e.lep THEN 'Yes' ELSE 'No' END),
+        (CASE WHEN e.elas_code = 'EO' THEN 'English Only'
+          WHEN e.elas_code = 'EL' THEN 'English Learner'
+          WHEN e.elas_code = 'IFEP' THEN 'Initial Fluent English Proficient'
+          WHEN e.elas_code = 'RFEP' THEN 'Reclassified Fluent English Proficient'
+          WHEN e.elas_code = 'TBD' THEN 'To Be Determined'
+          ELSE '' END),
```
Comment (malaffoon) [inline report-processor/src/main/resources/application.sql.yml]: How is the translation of this stuff being handled?

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 4]
```diff
@@ -772,6 +772,7 @@
     "assessment-session-id": "TestSessionId",
     "assessment-type": "AssessmentType",
     "claim": "Claim",
+    "elas": "ELAS",
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: Is this the header for the CSV? I wonder if we should expand it to `EnglishLanguageAcquisitionStatus`?

File: report-processor/src/main/resources/application.sql.yml (hunk 3 above)
Comment (dsnyder-fairway) [reply report-processor/src/main/resources/application.sql.yml]: It doesn't look like there is any support for translations on District / School Export

File: webapp/src/main/webapp/src/assets/i18n/en.json (hunk 4 above)
Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: +1 I'll update this

File: report-processor/src/main/resources/application.sql.yml (hunk 3 above)
Comment (dsnyder-fairway) [reply report-processor/src/main/resources/application.sql.yml]: done (also changed header to `EnglishLanguageAcquisitionStatus` here too)

File: report-processor/src/main/resources/application.sql.yml [hunk 5]
```diff
@@ -190,6 +191,7 @@ sql:
         (CASE WHEN e.section504 = 1 THEN 'Yes' when e.section504 = 0 THEN 'No' ELSE '' END),
         (CASE WHEN e.iep THEN 'Yes' ELSE 'No' END),
         (CASE WHEN e.lep THEN 'Yes' ELSE 'No' END),
+        COALESCE(e.elas_code, ''),
 
         (CASE WHEN EXISTS (SELECT 1 from student_ethnicity WHERE student_id = st.id AND ethnicity_code = 'HispanicOrLatinoEthnicity') THEN 'Yes' ELSE 'No' END ),
         (CASE WHEN EXISTS (SELECT 1 from student_ethnicity WHERE student_id = st.id AND ethnicity_code = 'AmericanIndianOrAlaskaNative') THEN 'Yes' ELSE 'No' END ),
```
Comment (wtritch) [inline report-processor/src/main/resources/application.sql.yml]: Oh shoot... are we double-defining our ethnicities here??  Aren't they client-specific?

**Pull request merged**